packages/http-client/src/http-client.ts

```diff
@@ -121,6 +123,61 @@ export function execute<ReturnT>(executeFn: ExecuteHttpRequestFn<ReturnT>) {
   };
 }
 
+function encodeRequestQueryParameters(
```

Comment:
Minor: Maybe "request is not needed?"
```suggestion
function encodeQueryParameters(
```